src/prometheus_handler.go

```diff
@@ -80,7 +87,7 @@ func makePromHistogram(label string, histogram map[string]int) string {
 	sort.Strings(bounds)
 
 	for _, bound := range bounds {
-		entry := fmt.Sprintf(`%s_bucket{le="%s"} %d`, label, bound, histogram[bound])
+		entry := fmt.Sprintf(`%s_bucket{le="%s", %s="%s"} %d`, label, bound, strings.TrimRight(labelname, ".bt"), datatype, histogram[bound])
```

Comment:
Can you make sure we maintain max 80 character length and if it exceed , move the statement to next line.